source/manages/self-hosted-billing.rst

```diff
@@ -0,0 +1,8 @@
+Self-hosted billing
+===================
+
+Mattermost Professional and Enterprise plans are offered as an annual subscription service.
+
+You'll be billed for registered users in your deployment. A registered user is a user who has an account and doesn't show as **Inactive** in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they're not deactivated in the billing period.
```

Comment:
@justinegeffen for this landing page, we need to mention the following things:

- Customers will have to buy a subscription and generate a license key with user seats equal to or greater than your current number of activated users (formerly called active users).
- Customers may add users at any time to their instance
- Customers are subject to quarterly true-up review (section 3.5 License True-up Review of [Software License Agreement](https://mattermost.com/enterprise-edition-terms/), also referenced in [docs](https://docs.google.com/document/d/1otVyzHWdR9MNyPRe6Yfu532-pGMLmdlTRaYzLoRdNgI/edit###%20In-product%20purchase%20experience%20for%20self-hosted%20%20https://docs.google.com/document/d/1otVyzHWdR9MNyPRe6Yfu532-pGMLmdlTRaYzLoRdNgI/edit#heading=h.c0fv1rt0bapi%20%20I've%20outlined%20the%20two%20things%20we%20exactly%20need%20for%20documentation)) to check for license seat count relative to number of activated users

> You'll be billed for registered users in your deployment. A registered user is a user who has an account and doesn't show as **Inactive** in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they're not deactivated in the billing period.

This isn't true for self-hosted